app/views/users/backup_code_setup/confirm_delete.html.erb

```diff
@@ -0,0 +1,23 @@
+<% title t('forms.backup_code.confirm_delete') %>
+
+<%= image_tag(asset_url('alert/warning-lg.svg'), alt: '', width: 54) %>
+
+<h1 class="h3 mb1 mt3 my0">
+  <%= t('forms.backup_code.confirm_delete') %>
+</h1>
+
+<div class="col-2 mb2">
+  <hr class="mt3 mb2 bw4 rounded border-yellow"></hr>
+</div>
+
+<p class="mb4">
+  <%= t('forms.backup_code.caution_delete') %>
```

Comment:
this one had `==` before, does it need to be marked as `raw`?

Reply:
The translations don't seem to have anything that needs to skip being escaped, so I didn't bring the `raw` part through:
https://github.com/18F/identity-idp/blob/master/config/locales/forms/en.yml#L16-L17